test/unit/listeners/substrate-listener/commands/orders/order-cancelled/order-cancelled.command.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { CommandBus, CqrsModule } from "@nestjs/cqrs";
+import { OrderStatus, TransactionLoggingService } from "../../../../../../../src/common";
+import { OrderCancelledCommand } from "../../../../../../../src/listeners/substrate-listener/commands/orders";
+import { Test, TestingModule } from "@nestjs/testing";
+import { escrowServiceMockFactory, transactionLoggingServiceMockFactory, createMockOrder, mockBlockNumber } from "../../../../../mock";
+import { OrderCancelledHandler } from "../../../../../../../src/listeners/substrate-listener/commands/orders/order-cancelled/order-cancelled.handler";
+import { EscrowService } from "../../../../../../../src/endpoints/escrow/escrow.service";
+
+describe("Order Cancelled Command Event", () => {
+  let orderCancelledHandler: OrderCancelledHandler;
+  let commandBus: CommandBus;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [
+        CqrsModule
+      ],
```

Comment:
Are you importing the `CqrsModule` because the handlers use a `CommandBus`? I think you should mock the `CommandBus` instead, refer to: https://github.com/debionetwork/debio-backend/blob/main/test/unit/listeners/substrate-listener/substrate-listener.handler.spec.ts

Reply:
In the meantime, please move the `commandBusMockFactory` from the `substrate-listener.handler.spec.ts` file to the global `mock.ts` file.

Reply:
Please run `npm run lint`. Remove any warnings if you find any.